### DISCUSSION TITLE
Deploying Rust to Vercel

### DISCUSSION THREAD

**Discussion by aleksrutins** [vercel/vercel#7227] [Help]
Hello,

I'm trying to deploy a Yew/Rust/WebAssembly app with a Rust API (through [vercel-community/rust](https://github.com/vercel-community/rust)), built with Trunk. However, when I run `vercel dev` (which I have set to `trunk serve`), it gives me a `could not determine executable to run` error, and when trying to deploy, it gives me `trunk: command not found`. Is there a way to deploy Rust/WebAssembly apps to Vercel?

Thanks!

**Comment by leerob** [comment]
The Rust community package is currently a WIP - we are open to contributions there to get it working! 😄

**Reply by aleksrutins** [reply]
The API doesn't seem to be having problems &mdash; what's happening is that the build step for the app itself (which uses Yew) is trying to go through NPM, which of course has no idea about either Rust or Trunk. I probably should have clarified that in the original question. Unless I'm misunderstanding how this works (which is quite likely), and the Rust runtime provides (or will provide) a build runtime alongside the API runtime.

**Reply by leerob** [reply]
Got it - yeah the Rust runtime is only for the API, not for the builder.
